src/Common/src/Interop/Linux/libc/Interop.inotify.cs

```diff
@@ -32,7 +32,7 @@ internal static int inotify_rm_watch(int fd, int wd)
                 }
                 else
                 {
-                    System.Diagnostics.Debug.Fail("inotify_rm_watch failed with " + hr);
+                    global::System.Diagnostics.Debug.Fail("inotify_rm_watch failed with " + hr);
```

Comment:
Are you also good with the longer names in Interop.Libraries? I think that's a good place to be explicit since it goes on its own line from the function name and parameters and actually represents the file name of the lib.

````
// Imports
internal partial class Interop {
    internal partial class Crypto {
            [DllImport(Interop.Libraries.SystemSecurityCryptographyNative)]
            internal static extern int GetX509Thumbprint(SafeX509Handle x509, byte[] buf, int cBuf); 
    }
    partial class Sys {
        [DllImport(Interop.Libraries.SystemNative, SetLastError=true, CharSet=CharSet.Ansi)]
        internal static extern int Stat(string path, out FileStatus output)]
    }
}

// Call sites
int ret = Interop.Sys.Stat(path, out status);
int ret = Interop.Crypto.GetX509Thumbprint(x509, buf, cBuf);
```